test/moves/whirlwind.test.ts

```diff
@@ -42,7 +42,7 @@ describe("Moves - Whirlwind", () => {
 
     await game.toEndOfTurn();
 
-    expect(staraptor.findTag((t) => t.tagType === BattlerTagType.MID_AIR)).toBeDefined();
+    expect(staraptor.hasTag(BattlerTagType.MID_AIR)).toBeTruthy();
```

Comment:
NIT:
```suggestion
    expect(staraptor).toHaveBattlerTag(BattlerTagType.MID_AIR);
```

I must have missed that on Temps PR